modules/kubernetes/fluxcd/main.tf

```diff
@@ -0,0 +1,100 @@
+terraform {
+  required_version = ">= 1.3.0"
+
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.23.0"
+    }
+    helm = {
+      source  = "hashicorp/helm"
+      version = "2.11.0"
+    }
+    flux = {
+      source  = "fluxcd/flux"
+      version = "1.4.0"
+    }
+    git = {
+      source  = "xenitab/git"
+      version = "0.0.3"
+    }
+  }
+}
+
+locals {
+  git_auth_proxy_url = "http://git-auth-proxy.git-auth-proxy.svc.cluster.local"
+  git_path_separator = var.git_provider.azure_devops != null ? "_git" : ""
+}
+
+resource "kubernetes_namespace" "git_auth_proxy" {
+  metadata {
+    name = "git-auth-proxy"
+    labels = {
+      "xkf.xenit.io/kind" = "platform"
+    }
+  }
+}
+
+resource "helm_release" "git_auth_proxy" {
+  depends_on = [
+    kubernetes_namespace.git_auth_proxy,
+    kubernetes_namespace.flux_system
+  ]
+
+  chart       = "oci://ghcr.io/xenitab/helm-charts/git-auth-proxy"
+  name        = "git-auth-proxy"
+  namespace   = kubernetes_namespace.git_auth_proxy.metadata[0].name
+  version     = "v0.8.1"
+  max_history = 3
+  values = [templatefile("${path.module}/templates/git-auth-proxy-values.yaml.tpl", {
+    git_provider = var.git_provider
+    bootstrap    = var.bootstrap
+    tenants      = var.namespaces
+  })]
+}
+
+resource "kubernetes_namespace" "flux_system" {
+  metadata {
+    name = "flux-system"
```

Comment:
We should add the labels for the namespace like the others.

Reply:
Nope. Flux generates a Namespace manifest as part of the bootstrap, which contains the labels.